index.d.ts

```diff
@@ -54,8 +98,39 @@ console.log(result);
 //=> ['https://sindresorhus.com/', 'https://avajs.dev/', 'https://github.com/']
 ```
 */
-export default function pMap<Element, NewElement>(
-	input: Iterable<Element>,
-	mapper: Mapper<Element, NewElement>,
-	options?: Options
-): Promise<NewElement[]>;
+declare const pMap: {
```

Comment:
Hello, I tried to do it this way on 941ed8b but TypeScript tests failed, so I reverted it via 31be4be, what do you think?

Reply:
The tests failed because the way the function is imported changed:

```diff
- import pMap from 'p-map';
- console.log(pMap, pMap.stop);
+ import pMap, {stop} from 'p-map';
+ console.log(pMap, stop);
```

Reply:
Per https://github.com/sindresorhus/p-map/commit/c9c088283e07280fb2c49ed4f83fd20cc0ad8b46, the symbol should be a named export called `pMapStop`.